regtest/README.md

```diff
@@ -1,6 +1,6 @@
 ## Regtest Mode
 WARNING Experimental!
-The CLI can work in regtest mode, by locally running a `zcashd` and `lightwalletd`.
+The CLI can work in regtest mode, by running a local `lightwalletd` that proxies to a local `zcashd` in "regtest mode".
```

Comment:
Proxies is a plural noun, so this sentence isn't quite right yet,

Reply:
I meant this:  https://www.wordhippo.com/what-is/the-verb-for/proxy.html